storagenode/storagenodedb/infodb.go

```diff
@@ -273,6 +274,59 @@ func (db *InfoDB) Migration() *migrate.Migration {
 					`CREATE INDEX idx_bandwidth_usage_created ON bandwidth_usage(datetime(created_at))`,
 				},
 			},
+			{
+				Description: "Create bandwidth_usage_rollup table.",
+				Version:     11,
+				Action: migrate.SQL{
+					`CREATE TABLE bandwidth_usage_rollups (
+										interval_start	TIMESTAMP NOT NULL,
+										satellite_id  	BLOB    NOT NULL,
+										action        	INTEGER NOT NULL,
+										amount        	BIGINT  NOT NULL,
+										PRIMARY KEY ( interval_start, satellite_id, action )
+									)`,
+				},
+			},
+			{
+				Description: "Convert bandwidth rollups created_at to UTC.",
```

Comment:
Just thinking out loud so I remember to follow up with this. We should probably verify that a long migration won't take down a production database during a deploy.